pkg/providers/amifamily/bootstrap/nodeadm.go

```diff
@@ -115,10 +115,8 @@ func (n Nodeadm) generateInlineKubeletConfiguration() (map[string]runtime.RawExt
 	if err != nil {
 		return nil, err
 	}
-	if len(n.Taints) != 0 {
-		kubeConfigMap["registerWithTaints"] = runtime.RawExtension{
-			Raw: lo.Must(json.Marshal(n.Taints)),
-		}
+	kubeConfigMap["registerWithTaints"] = runtime.RawExtension{
```

Comment:
Why are we taking it out of the len check now?

Reply:
There will always be at least the `unregistered` taint.